catalog/solr/catalog-solr-commands/pom.xml

```diff
@@ -15,12 +15,12 @@
 <project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
     <modelVersion>4.0.0</modelVersion>
     <parent>
-        <artifactId>core</artifactId>
-        <groupId>ddf.catalog.core</groupId>
+        <artifactId>catalog-solr</artifactId>
+        <groupId>ddf.catalog.solr</groupId>
```

Comment:
❓ Now that group IDs are changing, besides updating dependent artifacts downstream (handled separately), is there any build-level or infrastructure configuration we will need to update?

Reply:
I don't think so for this module since no one should have depended on these coordinates directly but would pull them in using the feature file which has the same coordinates as before.  

The impact is to anyone downstream using the `CachePutPlugin`.  The API coordinates and package did change.  But this interface is not whitelisted.